docs/content/latest/yugabyte-platform/secure-universes/enable-encryption-at-rest.md

```diff
@@ -0,0 +1,122 @@
+
+## Enable encryption at rest on an existing universe
+
+To enable encryption at rest on an existing universe, follow these steps:
+
+1. Open the Yugabyte Platform console and click **Universes**.
+2. Select the universe you want to enable.
+3. Click the **More** drop-down list and select **Manage Encryption Keys**. The **Manage Keys** dialog appears.
+4. Select **Enable Encryption-at-Rest for <UNIVERSE_NAME>**. When enabled, the **Key Management Service Config** option appears.
+5. Select your KSM configuration from the **Key Management Service Config** drop-down list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-configuration).
+6. Verify that the `EnableEncryptionAtRest` task completed successfully.
+
+## Back up and restore data from an encrypted at rest universe
+
+When backing up a cluster, you might restore the data to a different cluster with a different universe key or it might not have encryption enabled. As a result, a backup must include the decrypted data keys for all files.
+
+YugabyteDB uses two options to achieve this:
+
+- **If KMS integration is enabled**, back up the master universe key registry, encrypted with a KMS key, as part of the backup with the encrypted data files. On restore, decrypt the registry with the KMS key, and the registry is populated into the master system catalog and used to read the backed-up data files.
+- **If KMS is not used**, back up just the encrypted data files. On restore, use an out of band method to replay the universe key history to repopulate the registry. Once this is done, the cluster will be able to read the restored data files.
+
+## Rotate the encryption at rest keys
+
+Enabling encryption and rotating a new key works in two steps:
+
+1. Add the new universe key ID and key data to all the in-memory state of masters.
+2. Issue a cluster configuration change to enable encryption with the new key.
+
+The cluster configuration change does the following:
+
+- Decrypts the universe key registry with the previous latest universe key.
+- Adds the new key to the registry.
+- Updates the cluster configuration with the new latest key ID.
+- Encrypts the registry with the new key.
+
+Once encryption is enabled with a new key, only new data is encrypted with this new key. Old data will remain unencrypted, or encrypted, with an older key, until compaction churn triggers a
+re-encryption with the new key.
+
+To rotate the universe keys, perform the following steps:
+
+1. Open the Yugabyte Platform console and go the the universe.
+2. Click the **More** drop-down list and select **Manage Encryption Keys**. The **Manage Keys** window appears.
+3. Select **Rotate Key?** and then click **Submit**.
```

Comment:
Due to a current UI bug, the kms configuration doesn't show up as pre-selected in the modal (it should though). Should we explicitly state that the user should make sure they have chosen the same kms configuration in the rotate key modal?

Reply:
Updated to include mention of verifying the correct KMS configuration is selected in that rotate key dialog.